torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -660,6 +661,24 @@ def rewrite_node(node):
                 else:
                     # Fallback: aten.reshape is more stable if the input nodes have changed
                     return g.call_function(torch.ops.aten.reshape.default, args, kwargs)
+            if node.target == torch.ops.aten.split.Tensor:
```

Comment:
@sjameelTT can you please take a look at this?
The expectation is that we can simply call ttnn.split here but you see we have to make rm input, calculate an arg, change an arg. 

Please review, maybe something can be removed and hopefully everything can get hidden in the op itself.